internal/services/chaosstudio/chaos_studio_capability_resource_test.go

```diff
@@ -0,0 +1,166 @@
+package chaosstudio_test
+
+// NOTE: this file is generated - manual changes will be overwritten.
```

Comment:
Is there a Pandora PR in flight to add the resource definition to generate this one?

Reply:
nope